buildkite/client/pipeline.go

```diff
@@ -236,3 +292,15 @@ query GetPipelineId($pipelineSlug: ID!) {
 
 	return idResponse.Pipeline.Id, nil
 }
+
+// getTeamIDFromTeamUUID returns a team ID associated with the given team's uuid.
+//
+// Warning: undocumented Buildkite feature: they store team ids as base64("Team---" + uuid).
+// Neither GraphQL not Rest HTTP Buildkite APIs don't allow ro retrieve a team by its UUID so we use
+// this technique that mimics to Buildkite behavior.
+//
+// TODO(oleg): migrate to use TeamIDs instead of TeamUUIDs.
+//  Jira ticket: https://canvadev.atlassian.net/browse/FEIN-695
+func (c *Client) getTeamIDFromTeamUUID(UUID string) string {
```

Comment:
Up to you but, could this be a private package function instead? It is not using any of the `Client` attributes so it feels more like a helper function than a struct method 😄

Reply:
Yeah, more like a helper. I'made this change to address @belugarciafava's comment here https://github.com/Canva/terraform-provider-buildkite/pull/6#discussion_r492428267. Belen, do you mind to turn this into a static function instead?😀

Reply:
@alloveras, we are planning to get rid of this method completely in a short period once https://canvadev.atlassian.net/browse/FEIN-695 will be done so it seems to be okay to keep it as it is now while keeping in mind we will change it soon

Reply:
I am fine as long as this functionality is extracted and can be identified easily in case buildkite changes their approach and we need to update it quicky :)